packages/react-mutation-mapper/src/component/lollipopMutationPlot/LollipopMutationPlot.tsx

```diff
@@ -556,10 +556,76 @@ export default class LollipopMutationPlot<
 
     @autobind
     private getSVG(): SVGElement {
-        let svg: SVGElement = $(this.divContainer).find(
+        let lollipopSvg: SVGElement = $(this.divContainer).find(
             '.lollipop-svgnode'
         )[0] as any;
-        return svg;
+
+        const SVG_NS = 'http://www.w3.org/2000/svg';
+        const TEXT_STYLE = 'font-size: 12px; font-family:Arial;';
+        const TRACK_INDENT = 16;
+        const SUBTRACK_INDENT = TRACK_INDENT + 12;
+
+        function createTextTag(textNode?: Text, x: number = 0, y: number = 0) {
```

Comment:
i would avoid the practice of defining functions inside of other functions/methods.   if you wanted to write a test for this for example, you could not do so.   just put it up at the top of this file or in a helper file which is associated with this file.

Reply:
again, this would allow you to export these functions.    also, defining function this way inhibits performance.  it's totally negligible here but best to avoid in general